Tests/SentryTests/Integrations/Performance/UIViewController/SentryUIViewControllerPerformanceTrackerTests.swift

```diff
@@ -399,28 +399,6 @@ class SentryUIViewControllerPerformanceTrackerTests: XCTestCase {
         XCTAssertEqual(tracer?.children.filter { $0.operation.contains("initial_display") }.count, 0, "Tracer must not contain a TTID span")
         XCTAssertEqual(tracer?.children.filter { $0.operation.contains("full_display") }.count, 0, "Tracer must not contain a TTFD span")
     }
-
-    func testSecondViewController() {
```

Comment:
IIUC this test only checks if the view controller has a tracker associated. This way of storing a reference to the TTD tracker is outdated, so I removed the test